### PR TITLE
Prepare for release of V2.1

### PR DESCRIPTION
**Pull request by OsirisTerje**
Prepared code for release of V 2.1.

Need to test the components before proceeding.

### CODE REVIEW

**Comment by CharliePoole** [comment]
Build failure is due to using a branch name that is not legal as part of a version pre-release tag. You can either not use such branch names or fix the script similar to the NUnit 3 adapter script. This won't stop releasing (non pre-release) version 2.1, but prevents the automatic publication of builds from working.

Normally, we don't add issues or PRs related to doing the release to the milestone, since we don't want to list them in the change log or release notes. That is, it looks a little odd if the list of changes for 2.1 includes "Prepare for 2.1."

File: src/NUnitTestAdapter/NUnitTestAdapter.csproj [hunk 1]
```diff
@@ -52,17 +52,17 @@
...
     </Reference>
-    <Reference Include="Mono.Cecil.Pdb">
-      <HintPath>..\..\packages\Mono.Cecil.0.9.6.1\lib\net35\Mono.Cecil.Pdb.dll</HintPath>
+    <Reference Include="Mono.Cecil.Pdb, Version=0.9.6.0, Culture=neutral, PublicKeyToken=0738eb9f132ed756, processorArchitecture=MSIL">
+      <HintPath>..\..\packages\Mono.Cecil.0.9.6.4\lib\net35\Mono.Cecil.Pdb.dll</HintPath>
     </Reference>
-    <Reference Include="Mono.Cecil.Rocks">
-      <HintPath>..\..\packages\Mono.Cecil.0.9.6.1\lib\net35\Mono.Cecil.Rocks.dll</HintPath>
+    <Reference Include="Mono.Cecil.Rocks, Version=0.9.6.0, Culture=neutral, PublicKeyToken=0738eb9f132ed756, processorArchitecture=MSIL">
+      <HintPath>..\..\packages\Mono.Cecil.0.9.6.4\lib\net35\Mono.Cecil.Rocks.dll</HintPath>
```
Comment (CharliePoole) [inline src/NUnitTestAdapter/NUnitTestAdapter.csproj]: Questions:

1. Why upgrade mono.cecil? Have we tested the new release?
2. If we are going to specify the exact version, why does the include line use 0.9.6.0 while the hint line uses 0.9.6.4?

Comment (OsirisTerje) [inline src/NUnitTestAdapter/NUnitTestAdapter.csproj]: 1.  I just noted it was a minor-minor update, so just did it.  Checked the adapter afterwards and all seems to work as it should.   A bit indifferent on this though - should I go back to 
2. This was just updated automatically by the Visual Studio Nuget manager.  Not sure at all why the numbers are as they are.   Different assembly and package versions perhaps?  SHould I just revert this?  I don't quite like staying on earlier releases, in particular when it is minor-minor (4th digit changes)

**Comment by OsirisTerje** [comment]
Right, I forgot about the / in the branch names, I just push the final one up to a non-slashed branch.    Cleared the milestone marking

**Comment by OsirisTerje** [comment]
Tested it now as:
VSIX on VS 2015 :  works
VSIX on VS 2017:  works
Nuget on VS 2015:  works
Nuget on VS2017 works

So I guess it is close to ready to be released.

**Comment by CharliePoole** [comment]
I agree it's ready to release.

**Comment by OsirisTerje** [comment]
Just noted the release notes in the docs repo for the adapter only refers to the V3 adapter.  Should we  update the nunit.org repo with the release notes for this V2 adapter?

**Comment by CharliePoole** [comment]
@OsirisTerje Actually, the release notes contain info for both adapters. I added the v2 info when we released 2.0 - see nunit/docs#89. You just have to scroll down to find 2.0. That might not be very convenient for users of the original adapter, so maybe we should add a heading for the old adapter and link to it from the top of the page.

After you and I decided to combine the docs, I created the outline and added the pages from the nunit3-vs-adapter repository. I created issues for you to merge in info from the existing v2 repository wiki: nunit/docs#83 nunit/docs#84 nunit/docs#85 nunit/docs#86 nunit/docs#87 nunit/docs#88 nunit/docs#89. Only the last one is done.

You may want to consider adding a copy of the release v2 release notes to the v2 repo until you are ready to complete the merge tasks.

Regarding nunit.org, I suppose we should either remove the adapter pages or update them. What do you think?

**Comment by OsirisTerje** [comment]
Regarding nunit.org, I agree, it is just confusing having them there now.

**Comment by CharliePoole** [comment]
OK, I'll create an issue for myself to remove those pages from nunit.org.

**Comment by OsirisTerje** [comment]
Just released the adapter, updated on nuget and VS gallery/marketplace.
I added the extra page for the V2 release notes under the adapter section.  

We now don't produce the combined package, so I added a note stating it is discontinued.   I see from the statistics there are 70K downloads last 6 weeks of the adapter itself (from nuget), and around 9K of the combined adapter/framework package.  For comparison, the V3 adapter on nuget has 116K downloads.

**Pull request merged**